.github/workflows/ci.yml

```diff
@@ -36,14 +38,11 @@ jobs:
       run: npm run lint
     - name: Test
       run: npm run test
-    - name: Coverage Report
-      uses: codecov/codecov-action@v3
-    - name: Preview Updated Versions (dry run)
+    - name: "[Lerna] Preview Updated Versions (dry run)"
       # switch from detatched head to a real branch so we can pass it to `--allow-branch`
       run: |
         git switch -c "actual-branch-not-detached-head"
-        echo "Installing commitlint"
-        npm install --save commitlint
-        npm install --save conventional-changelog-conventionalcommits
-        echo "Finished installing Installing commitlint"
-        lerna version --allow-branch actual-branch-not-detached-head --no-git-tag-version --no-changelog --no-push --yes
+        npx lerna init 
```

Comment:
I believe `lerna init` is typically only run to generate the `lerna.json` configuration file. Out of curiosity, does this work without `lerna init`? If not, are there changes generated from this `lerna init` command that should be actually persisted/committed in the `lerna.json` file instead?

Reply:
This was me not fully understanding npx, npm and lerna. I've read the lerna docs and understand better now what this does. Updated in latest PR.